snippets/operator-not-available.adoc

```diff
@@ -0,0 +1,9 @@
+// When including this file, ensure that {operator-name} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+{operator-name} is not available yet for {product-title} {product-version}. The Operator is planned to be released in the near future.
```

Comment:
Tiny nit. _not available yet for_ sounds a bit  awkward. Maybe _not yet available for_ or _not currently available for_ ??

Reply:
Good suggestion, I went with "not currently available"